alembic/util/pyfiles.py

```diff
@@ -45,6 +46,12 @@ def template_to_file(
             "template-oriented traceback." % fname
         )
     else:
+        if append and append_delimiter:
```

Comment:
can we simplify this

append use case here is line-based and is only used by command , and we also know the file exists and has text in it if append was sent, so:

```diff
diff --git a/alembic/util/pyfiles.py b/alembic/util/pyfiles.py
index fc9ef6e..67c5396 100644
--- a/alembic/util/pyfiles.py
+++ b/alembic/util/pyfiles.py
@@ -46,6 +46,8 @@ def template_to_file(
         )
     else:
         with open(dest, "ab" if append else "wb") as f:
+            if append:
+                f.write(b"\n\n")
             f.write(output)

```

Reply:
Yes, that is preferable.  I will update.  I didn't know the internal usage, so could only hope the library operated like that.